nuxt.config.ts

```diff
@@ -226,23 +224,17 @@ const config: NuxtConfig = {
       },
     ],
     friendlyErrors: false,
-    postcss: {
-      plugins: {
-        tailwindcss: {},
-        autoprefixer: {},
-        'postcss-focus-visible': {},
-      },
+  },
+  postcss: {
```

Comment:
Is postCSS currently not working because the settings are incorrect?

Reply:
I'm not totally sure. For me the dev server currently crashes on an error about not being able to find a specific template file in the `.nuxt` directory.

Is that what you're seeing as well? I didn't see any postCSS errors specifically.

Reply:
I haven't tried this branch yet. But I did try to update locally before (when I saw the bridge beta notice in the `pnpm dev` output), and had that template not found error. It's weird because we don't set the path in the options that Nuxt is trying to use, and I don't know where it's coming from.

Reply:
It's internal Nuxt template stuff. I think it is looking for app.template.html but for some reason using document.template.html for it. I will open an issue in nuxt/bridge repository later today for it.